etc/links.profile

```diff
@@ -0,0 +1,51 @@
+# Firejail profile for elinks
+# Description: Text WWW browser
+# This file is overwritten after every install/update
+#
+# Persistent local customizations
+include links.local
+# Persistent global definitions
+# include globals.local
+
+blacklist /tmp/.X11-unix
+
+noblacklist ${HOME}/.links
+
+include disable-common.inc
+include disable-devel.inc
+include disable-exec.inc
+include disable-interpreters.inc
+include disable-passwdmgr.inc
+include disable-programs.inc
+include disable-xdg.inc
+
+mkdir ${HOME}/.links
+whitelist ${HOME}/.links
+whitelist ${DOWNLOADS}
+include whitelist-common.inc
+include whitelist-var-common.inc
+
+caps.drop all
+ipc-namespace
+machine-id
+netfilter
+no3d
+nodvd
+nogroups
+nonewprivs
+noroot
+nosound
+notv
+nou2f
+novideo
+protocol unix,inet,inet6
+seccomp
+shell none
+tracelog
+
+disable-mnt
+private-cache
+private-dev
+private-tmp
```

Comment:
Also try this please: `private-etc ca-certificates,crypto-policies,resolv.conf,ssl`

Reply:
This is often too limiting for associated programs (e. g. `mpv` was unable to start)

Reply:
we usually don't write profiles that allow other programs to work
see https://github.com/netblue30/firejail/issues/1718#issuecomment-406819438

Reply:
Better: `private-etc ca-certificates,crypto-policies,nsswitch.conf,pki,resolv.conf,ssl` and if e.g. exists /etc/links also links.

Reply:
> we usually don't write profiles that allow other programs to work
> see [#1718 (comment)](https://github.com/netblue30/firejail/issues/1718#issuecomment-406819438)
> and
> [#1770 (comment)](https://github.com/netblue30/firejail/issues/1770#issuecomment-364498100)

hmm... if this is the case then the whole `novideo`, `noaudio`, `x11` discussion is pointless, right?

Reply:
private-etc: for internet: `ca-certificates,crypto-policies,nsswitch.conf,pki,resolv.conf,ssl` and if you want to play sound: `alsa,asound.conf,machine-id,openal,pulse` and if you want to open GUIs: `fonts`

Reply:
Also `include disable-programs.inc` can prevent custom programs from starting.

Reply:
@jose1711 were you able to try the private-etc line @rusty-snake provided? 
`private-etc ca-certificates,crypto-policies,nsswitch.conf,pki,resolv.conf,ssl`

Reply:
But then also add `alsa,asound.conf,machine-id,openal,pulse` for sound and `fonts` for GUIs (xlinks) and `alternatives`.

Reply:
@rusty-snake how about doing this:
```
private-etc ca-certificates,crypto-policies,nsswitch.conf,pki,resolv.conf,ssl
# Uncomment the following line to allow external media players
# private-etc alsa,asound.conf,alternatives,machine-id,openal,pulse
```

Reply:
@Fred-Barclay sound good, but
1. `# Uncomment the following line (or put it in your links.local) …`
2. On Fedora: `ll /bin/links`: `/bin/links -> /etc/alternatives/links`
3. xlinks will need `fonts`

Reply:
@rusty-snake I don't understand 2.

Reply:
added `private-etc` + comment on how to enable media players

Reply:
@Fred-Barclay we need alternatives, because on some system is links a symlink to /etc/alternativs/links